stock_picking_track/stock_picking_track.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Addons modules by CLEARCORP S.A.
+#    Copyright (C) 2009-TODAY CLEARCORP S.A. (<http://clearcorp.co.cr>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields
+import openerp.addons.decimal_precision as dp
+
+class stock_move_thread(models.Model):
+    
+    _inherit = ['stock.move', 'mail.thread']
+    _name = "stock.move"
+    _description = "Stock Move"
+    _order = 'date_expected desc, id'
+    _log_create = False
+    
+    product_id= fields.Many2one('product.product', 'Product', required=True, select=True, domain=[('type', '<>', 'service')], states={'done': [('readonly', True)]}, track_visibility='onchange')
```

Comment:
This line needs fixing there is no need to overwrite all the attributes of the *Many2one* field.

```python
product_id = fields.Many2one(track_visibility='onchange')
```